cmd/argo/commands/server.go

```diff
@@ -191,18 +190,8 @@ See %s`, help.ArgoServer),
 		},
 	}
 
-	defaultBaseHRef := os.Getenv("BASE_HREF")
```

Comment:
Please correct me if I'm wrong.
I think we want to keep this right? but use `defaultBaseHRef := os.Getenv("ARGO_BASE_HREF")` instead?
otherwise setting environmental variable `ARGO_BASE_HREF` wouldn't have an effect

Reply:
See the `viper` usage [below](https://github.com/argoproj/argo-workflows/blob/8571e1f1e8b78776e02a5d48013dd3c1f4ecd853/cmd/argo/commands/server.go#L215) that. That was added in #6767 as I referenced in the PR description. This is listed [in the docs](https://github.com/argoproj/argo-workflows/blob/66680f1c9bca8b47c40ce918b5d16714058647cb/docs/environment-variables.md?plain=1#L140) as well